src/main/java/uk/gov/justice/digital/hmpps/keyworker/model/KeyworkerStatus.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.hmpps.keyworker.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum KeyworkerStatus {
+    ACTIVE("ACT"),
+    INACTIVE("INA"),
+    UNAVAILABLE_ANNUAL_LEAVE("UAL"),
+    UNAVAILABLE_LONG_TERM_ABSENCE("ULT"),
+    UNAVAILABLE_NO_PRISONER_CONTACT("UNP"),
+    UNAVAILABLE_SUSPENDED("USU");
+
```

Comment:
Just wondering if the API should master the status descriptions also (instead of the UI),  in case we eventually switch to a reference data table.